lms/resources/_js_config/__init__.py

```diff
@@ -87,6 +87,12 @@ def add_document_url(self, document_url):
                     _query={"document_url": document_url},
                 ),
             }
+        elif document_url.startswith("canvas://"):
+            course_id = self._request.params["custom_canvas_course_id"]
+            file_id = self._request.params["file_id"]
+            resource_link_id = self._request.params["resource_link_id"]
+
+            self.add_canvas_file_id(course_id, resource_link_id, file_id)
```

Comment:
This is now the only place where `add_canvas_file_id()` is called so you could inline it, but I see that in https://github.com/hypothesis/lms/pull/3186/ you've deleted it